packages/core/admin/admin/src/content-manager/components/DynamicTable/BulkActionsBar/tests/index.test.js

```diff
@@ -78,10 +78,34 @@ describe('BulkActionsBar', () => {
     });
 
     await act(async () => {
-      await fireEvent.click(screen.getByRole('button', { name: /\bDelete\b/ }));
-      fireEvent.click(screen.getByRole('button', { name: /confirm/i }));
+      await user.click(screen.getByRole('button', { name: /\bDelete\b/ }));
+      await user.click(screen.getByRole('button', { name: /confirm/i }));
     });
 
     expect(mockConfirmDeleteAll).toHaveBeenCalledWith([]);
   });
+
+  it('should show publish modal if publish button is clicked', async () => {
+    const onConfirmPublishAll = jest.fn();
+    setup({ showPublish: true, onConfirmPublishAll });
+
+    await act(async () => {
+      await user.click(screen.getByRole('button', { name: /\bpublish\b/i }));
+      await user.click(screen.getByTestId('confirm-publish'));
+    });
```

Comment:
I find it weird that wrapping in act is necessary in these tests to get rid of the warning. We don't have that issue in the marketplace tests for example, though they have a lot more user interactions